Update perform-migration-from-svn-to-git.md (#12211)
MicrosoftDocs/azure-devops-docs: docs/repos/git/perform-migration-from-svn-to-git.md

>
> This command can take a few minutes to several hours depending on the size of the SVN repository. Upon completion, you will have a Git checkout of your repository.

### Migrate only specific revisions

> When not specified, git-svn clone will migrate all the revisions from first commit (r1) to HEAD. Should there any case arise, that one needs to migrate only specific set of revisions, the command for git-svn clone, should be appended with an option of -r.

Reviewer: ```suggestion
When not specified, `git-svn clone` will migrate all of the revisions from the first commit (r1) to HEAD. If you need to migrate only a specific set of revisions, the command for `git-svn clone` should be appended with an option of `-r`.
```
